src/test/java/com/training/sample/tests/SampleGetJson.java

```diff
@@ -0,0 +1,24 @@
+package com.training.sample.tests;
```

Comment:
Since this is a utility of this framework, This test should be placed in - test/java/com/training/utilities/tests/
and all the implementation should go in - main/java/com/training/utilities/